.github/workflows/ci.yml

```
@@ -46,7 +50,7 @@ jobs:
       - name: Install Dependencies
         run: yarn install --frozen-lockfile
       - name: Run Tests
-        run: yarn test:ember
+        run: yarn test
```

```suggestion
        run: yarn test:ember
        working-directory: ./test-app
```

Like this we won't need to run the lint job twice